<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContinuousSnapshotMoviePropertyPage</class>
 <widget class="QWizardPage" name="ContinuousSnapshotMoviePropertyPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Movie Properties</string>
  </property>
  <property name="subTitle">
   <string>Please specify the properties of output files.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1,0">
   <item>
    <widget class="QCheckBox" name="movieCheckBox">
     <property name="text">
      <string>Output movie files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="filenameGroupBox">
     <property name="title">
      <string>File name</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="filenameTableWidget">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
        </property>
        <property name="columnCount">
         <number>0</number>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>16</number>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Play speed</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QRadioButton" name="lengthRadioButton">
          <property name="text">
           <string>Specify movie length[sec]</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="lengthSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>3600</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="fpsRadioButton">
          <property name="text">
           <string>Specify frames per second</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="fpsSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="profileLabel">
           <property name="text">
            <string>Profile:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="profileComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>movieCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>filenameTableWidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>movieCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>profileComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lengthRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>lengthSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fpsRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>fpsSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
